02-intermediate-ruby/object-inheritance.md

```diff
@@ -3,77 +3,131 @@
 - Describe objects in terms of _is-a_ relationships
 - Use _inheritance_ to indicate a hierarchy of ownership or lineage between classes
 - Explore overriding method definitions in _subclasses_
-- Use `super` to invoke the behavior of a _base class_ in a _subclass 
+- Use `super` to invoke the behavior of a _base class_ in a _subclass
 
 ## Object Inheritance: _is-a_
 _Inheritance_ is used to model the behaviors of one class after another. We would want to do this when two objects have a clear chain of ownership. Often the parent class, or _base class_, is more general while the child, or _subclass_, is more specific.
 
-For example, we could have a `Bird` class that describes the general behaviors of birds. This is a _base class_. We could create `Penguin` and `Hawk` classes that model more specific behaviors. A `Penguin` is very different than a `Hawk`, but they are both birds. It would make sense then, to declare them _subclasses_ of `Bird`. 
+For example, we could have a `LibraryItem` class that describes the general behaviors of items in a library. This is a _base class_ or _superclass_. We could create `Book` and `Video` classes that model more specific behaviors. A `Book` is very different than a `Video`, but they are both items in the library. It would make sense then, to declare them _subclasses_ of `LibraryItem`.
```

Comment:
I really like that you're referencing two different objects that could inherit this behavior. Could we include Video in the code sample and diagram as well? Just in the first set of things, then in the later more complex examples, we could say that we are just going to focus on Book and leave off Video for now.